eng/pipelines/eng-common-sync.yml

```diff
@@ -174,5 +174,17 @@ stages:
                     arguments: >
                       -PRDataArtifactPath "$(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{ parameters.PRDataFileName }}"
                       -AuthToken "$(azuresdk-github-pat)"
-                      -devOpsLogging
+                    pwsh: true
+
+                - task: PowerShell@2
```

Comment:
What value to you think we have by adding the auto-merge label here? Is it only to prevent someone from needing to click the merge button themselves?

Reply:
We gain some speed in coordinating the merge. If the Sync PRs are merged then we want to merge the Tools PR and soon as is possible. The auto-merge will help in the case the user forgets and the PR is just waiting.